x/concentrated-liquidity/lp.go

```diff
@@ -15,6 +15,16 @@ import (
 
 const noUnderlyingLockId = uint64(0)
 
+// CreatePositionData reprsents the return data from CreatePosition.
```

Comment:
```suggestion
// CreatePositionData represents the return data from CreatePosition.
```

interesting the spell check linter didn't catch that